packages/flutter/lib/src/widgets/editable_text.dart

```diff
@@ -1564,6 +1608,33 @@ class EditableText extends StatefulWidget {
   /// {@macro flutter.services.TextInputConfiguration.enableIMEPersonalizedLearning}
   final bool enableIMEPersonalizedLearning;
 
+  /// {@template flutter.widgets.editableText.contentInsertionMimeTypes}
+  /// Used when a user inserts image-based content through the device keyboard,
+  /// currently only used on Android.
+  ///
+  /// The passed list of strings will determine which MIME types are allowed to
+  /// be inserted via the device keyboard. If no `onContentInserted` callback
+  /// is provided, this field will be ignored.
+  ///
+  /// {@tool snippet}
+  /// The default mime types are given by [kDefaultContentInsertionMimeTypes]. These are all the mime types
+  /// that are able to be handled and inserted from keyboards.
+  ///
+  /// {@end-tool}
```

Comment:
Did you accidentally make this a snippet?

Reply:
Yep, it was supposed to be but then I removed some code example and forgot to change it.